### DISCUSSION TITLE
HelloGitHub 徽章计划

### DISCUSSION THREAD

**Discussion by 521xueweihan** [Zxilly/go-size-analyzer#104] [General]
🎉 恭喜！您的[项目](https://hellogithub.com/repository/c31575032ae144c6b22c553399a42742)已被 HelloGitHub 社区收录和推荐，现邀请您加入 HelloGitHub 徽章计划。加入后，您将获得以下特权：

- 社区认可: 徽章代表你的项目已通过 HelloGitHub 社区的严格筛选，并获得推荐。
- 提高曝光: 佩戴徽章后，你的项目将获得更多推荐流量，吸引更多用户和贡献者。
- 增强互动: 用户可通过徽章快速了解项目，并与之互动（点赞、评论、收藏）。
- 收集反馈: 获取广大用户的真实反馈，持续优化你的项目。
- 特殊标识: 认证后，你的评论将有显著标识，并获得置顶优先权。

📌 [立即点击](https://hellogithub.com/repository/c31575032ae144c6b22c553399a42742/embed) 完成徽章佩戴，加入 HelloGitHub 徽章计划，让你的开源项目绽放更耀眼的光彩。

---

**HelloGitHub**，一个专注于发现、分享、推广开源项目的社区。自 2016 年创立以来，从每月一更的月刊发展到今天，已经成为一个拥有超过 **1 万** 用户的充满活力的开源社区。我们的足迹遍布多个内容平台，并获得了全网 **50 万** 粉丝的信赖与支持。

**Comment by Zxilly** [comment]
有小一点的badge版本吗

**Reply by 521xueweihan** [reply]
容我开发一下，等我弄完再来。

**Reply by 521xueweihan** [reply]
@Zxilly  您好，小徽章功能已上线，可通过上面的认领地址进行认领和佩戴。

<img width="693" alt="截屏2024-07-04 16 00 22" src="https://github.com/Anxcye/anx-reader/assets/8255800/2bdbd20e-f6ea-4900-9366-737672d38a74">

**Reply by Zxilly** [reply]
https://github.com/Zxilly/go-size-analyzer/commit/08be777310addee76e43bfa44147fd42061acf0b

**Reply by 521xueweihan** [reply]
@Zxilly 为啥 README.md 不佩戴下，HelloGitHub 正在做国际化😅

**Reply by Zxilly** [reply]
现在看来还是只有中文吧，而且项目描述应该都是单语言的？

**Reply by 521xueweihan** [reply]
全站都会做国际化，包括项目项目描述。

**Reply by Zxilly** [reply]
那等实现了以后再加上吧

**Reply by 521xueweihan** [reply]
好的

**Reply by 521xueweihan** [reply]
@Zxilly 
<img width="790" alt="image" src="https://github.com/user-attachments/assets/103b72b9-f9b3-4431-b32a-652efb756bac">

您好，网站的国际化已上线，欢迎佩戴HelloGitHub徽章～
